src/main/java/pro/sky/course_work2kurs/controller/ExamController.java

```diff
@@ -0,0 +1,26 @@
+package pro.sky.course_work2kurs.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import pro.sky.course_work2kurs.model.Question;
+import pro.sky.course_work2kurs.service.ExaminerServiceImpl;
+
+import java.util.Collection;
+import java.util.Collections;
+
+@RestController
+@RequestMapping("/exam")
+public class ExamController {
+    private final ExaminerServiceImpl examinerService;
+
+    public ExamController(ExaminerServiceImpl examinerService) {
```

Comment:
```suggestion
    public ExamController(ExaminerService examinerService) {
```
по интерфейсу инъекцию надо делать, говорил про это на разборе